freezeyt/freezer.py

```diff
@@ -239,12 +239,28 @@ class Freezer:
     saver: Saver
 
     def __init__(self, app, config):
-        self.app = app
         self.config = config
         self.check_version(self.config.get('version'))
 
         self.freeze_info = hooks.FreezeInfo(self)
 
+        module_name = config.get('module_name')
+        if app is None:
+            if module_name is None:
```

Comment:
There is a duplication to cli.py, which we test same case. Here we test API of freeze in cli.py we test inputs to command line.